clang/test/Sema/builtin-assume-aligned.c

```diff
@@ -74,7 +74,7 @@ int test13(int *a) {
 }
 
 int test14(int *a, int b) {
-  a = (int *)__builtin_assume_aligned(b, 32); // expected-error {{incompatible integer to pointer conversion passing 'int' to parameter of type 'const void *}}
+  a = (int *)__builtin_assume_aligned(b, 32); // expected-error {{non-pointer argument to '__builtin_assume_aligned' is not allowed}}
```

Comment:
Can we add the test cases from the issue, specifically: https://github.com/llvm/llvm-project/issues/110914#issuecomment-2390964984 and https://github.com/llvm/llvm-project/issues/110914#issuecomment-2390985949

We should always include tests that trigger crashes we are fixing to catch possible future regression.

Reply:
Done